src/containers/Login.vue

```diff
@@ -29,7 +29,7 @@
                   class="email-button email-button--top"
                   variant="primary"
                 >
-                  ACCESS BETA
+                  GET STARTED
```

Comment:
The font sizes on the landing page seem to be a bit different than the ones on Figma, especially for this button. Could you update them to make sure they match?